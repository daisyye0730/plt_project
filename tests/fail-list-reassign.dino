int main() {
    List(int, 3) li;
    int a;

    a = -1;
    li = [2, 3, 4];
    li['c'] = a;
    print(li[5]);

}