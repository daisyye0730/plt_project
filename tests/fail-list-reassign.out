Fatal error: exception Failure("illegal access List(int, 3)[char] in li[c]")
